src/players/VAST.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import videojs from 'maestro-videojs-vast';
+import 'maestro-videojs-vast/dist/maestroVideoJsVast.css';
+import { callPlayer } from '../utils';
+import createSinglePlayer from '../singlePlayer';
+
+const PLACEHOLDER_VIDEO = 'https://storage.googleapis.com/maestro_video/blank.mp4';
+export class VAST extends Component {
+  static displayName = 'VAST';
+
+  static canPlay = url => typeof url === 'string';
+
+  static loopOnEnded = false;
+
+  callPlayer = callPlayer;
+
+  load(url) {
+    if (this.container) {
+      this.player = videojs(this.container, {
+        autoplay: false,
+        preload: false,
+      });
+      this.player.ads();
+      this.player.vast({ url });
+
+      const onAdStart = () => {
+        this.props.onDuration(this.getDuration());
+        this.props.onStart();
+      };
+
+      const removeListeners = () => {
+        this.player.off('adsready', this.props.onReady);
+        this.player.off('adstart', onAdStart);
+        this.player.off('play', this.props.onPlay);
```

Comment:
all this stuff is for the empty mp4 right? we likely want to end if the mp4 starts right?

Reply:
yeah exactly. naturally videojs and it's ad's plugin will start playing the blank video, but it won't start playing without a video to play (like you can't start videojs with a vast and not a video), however it will try its hardest to start the blank video after the VAST has ended, so this is the hack to get around that.